pkgs/applications/networking/mailreaders/mswatch/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchzip
+, pkg-config
+, autoreconfHook
+, bison
+, flex
+, glib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mswatch";
+  # Stable release won't compile successfully
+  version = "unstable-2018-11-21";
+
+  src = fetchzip {
+    url = "https://sourceforge.net/code-snapshots/svn/m/ms/mswatch/code/mswatch-code-r369-trunk.zip";
+    hash = "sha256-czwwhchTizfgVmeknQGLijYgaFSP/45pD2yhDKj5BKw=";
+  };
```

Comment:
This link is only valid for a very short time.
sourceforge removes this snapshot again.

After which someone needs to manually re-trigger and re-generate that snapshot via the "Download Snapshot" link at https://sourceforge.net/p/mswatch/code/369/tree/

Which means, this is not suitable for nixpkgs.

Reply:
> This link is only valid for a very short time. sourceforge removes this snapshot again.
> 
> After which someone needs to manually re-trigger and re-generate that snapshot via the "Download Snapshot" link at [sourceforge.net/p/mswatch/code/369/tree](https://sourceforge.net/p/mswatch/code/369/tree/)
> 
> Which means, this is not suitable for nixpkgs.

Are you sure about that? I ran:

```
git grep sourceforge $(git grep -l unstable)
```

And I saw more packages that use similar URLs... Also, now - more then a week after the link was generated, it still works for me.

Reply:
The link works, because I did the thing and re-generated that snapshot via the link above when I wrote my review.
I got http/404s before that and was very confused.

I can't say for sure if that's just a one-off error.
But the two other packages[^1][^2] using `https://sourceforge.net/code-snapshots/*` are not `svn`, so this might very well be specific to `svn`.

[^1]: https://github.com/NixOS/nixpkgs/blob/b0fd86580bff587d81f6d2563edaebc80ac11e01/pkgs/tools/graphics/enblend-enfuse/default.nix#L24
[^2]: https://github.com/NixOS/nixpkgs/blob/2b73d5c230df157acc8498e033a55978586af052/pkgs/applications/misc/audio/sox/default.nix#L43